airflow/executors/celery_executor.py

```diff
@@ -274,6 +330,61 @@ def execute_async(self,
     def terminate(self):
         pass
 
+    def try_adopt_task_instances(self, tis: List[TaskInstance]) -> List[TaskInstance]:
+        # See which if the tis are still alive (or have finished even!)
```

Comment:
`see which if the tis...` may contain a typo?

Reply:
Yup, `which of` it should be